submissions/modupeAdeOnojobi/dom/product.js

```diff
@@ -0,0 +1,43 @@
+let quantity = document.getElementById('quantity');
+let price = document.getElementById('price');
+let shippingCost = document.getElementById('shipping-cost');
+let cost = document.getElementById('total-cost');
+let button = document.getElementById('pricing-btn');
+let sumPricePrice = Number(price.textContent) * quantity.value;
```

Comment:
I think you wanted to write sumPrice here and not sumPricePrice

Reply:
It has been corrected.